### PR TITLE
Dont enable pixel when disabled option is enabled

### PR DESCRIPTION
**Pull request by joachimhviid**

### CODE REVIEW

File: lib/templates/plugin.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class TikTok {
 
 export default (ctx, inject) => {
   const instance = new TikTok(<%= JSON.stringify(options) %>);
-  instance.enable();
+  if (!instance.options.disabled) instance.enable();
```
Comment (GuillaumeFerron) [inline lib/templates/plugin.js]: Thanks for this, missed it

Could you just use the brackets writing, and include the whole of the closure in it?

I'll approve then

**Review by GuillaumeFerron — CHANGES_REQUESTED** [review]

**Comment by joachimhviid** [comment]
Let me know if I misunderstood what you meant :smile:

File: lib/templates/plugin.js [hunk 2]
```diff
@@ -16,7 +16,9 @@ class TikTok {
 
 export default (ctx, inject) => {
   const instance = new TikTok(<%= JSON.stringify(options) %>);
-  instance.enable();
+  if (!instance.options.disabled) {
```
Comment (GuillaumeFerron) [inline lib/templates/plugin.js:19]: Can you put l18, 22, 23 and 24 in the if statement?
Thanks!

**Review by GuillaumeFerron — CHANGES_REQUESTED** [review]

File: lib/templates/plugin.js (hunk 2 above)
Comment (joachimhviid) [reply lib/templates/plugin.js:19]: Hmm, I don't think that would work as you would then be unable to call the `enable` function, which you would still want to do once consent has been given.

Comment (GuillaumeFerron) [reply lib/templates/plugin.js:19]: You are right, thanks for the note, I am in between projects now, not focused enough ;)

**Pull request merged**